uefi-raw/Cargo.toml

```diff
@@ -23,5 +23,9 @@ bitflags.workspace = true
 ptr_meta.workspace = true
 uguid.workspace = true
 
+[features]
```

Comment:
There's no error type in uefi-raw now, so I think the `features` section can be dropped